packages/utils/src/analytics.js

```diff
@@ -10,4 +10,23 @@ const sendEnterpriseTrackEvent = (enterpriseUUID, eventName, properties = {}) =>
   );
 };
 
-export default sendEnterpriseTrackEvent;
+// How long to delay an event, so that we allow enough time for any async analytics event call to resolve.
+// https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/
+const CLICK_DELAY_MS = 300; // 300ms replicates Segment's ``trackLink`` function
```

Comment:
What do you think about also exporting this as a constant for consumers to pull in if needed (e.g., if a test needs to force a delay for whatever reason that matches this 300ms).